pubsub/google/cloud/pubsub_v1/subscriber/policy/thread.py

```diff
@@ -139,9 +139,12 @@ def on_response(self, response):
 
         For each message, schedule a callback with the executor.
         """
+        def callback_completed(future):
+            logger.debug('Result: %s' % future.result())
```

Comment:
1. `logger.debug('Result: %s', future.result())` (i.e. `logger.debug` will format for you)
1. Can you define this at the module level? It looks like a closure but doesn't enclose any scope.
1. Otherwise LGTM, though maybe the tests should be updated? (I'll defer to @lukesneeringer on that).

Reply:
1. Gah, yeah, will change that.  I simply copied the line of code from the previous location :)
2. Will do
3. Wasn't really sure of the best way to test for a debug statement being output.  If there are constructs/patterns in place that test for logging output (using mocks, whatever), I'll gladly add one for this.

Reply:
No worries about 1, sometimes its "you break it you buy it".

@lukesneeringer Care to chime in on 3?

Reply:
Made the first two fixes, squashed the two commits into one.  Let me know if that's not preferred.  Thanks.

Reply:
@jeffkpayne Thanks for being considerate of our history in `master`. I used to do that as well, but generally it makes review harder since the old and the new become interspersed. On this PR it's irrelevant (because the PR is so small) but it's worth knowing. Also, note that GitHub has a "merge squash" button (which we use), so your need to squash is no longer a thing.

Reply:
Thanks for the feedback.  Noted!